tests/test_landing_pad_tracking.py

```diff
@@ -0,0 +1,310 @@
+
+        assert actual == expected
+
+    def test_is_similar_negative_more_than_threshold(self):
+        """
+        Test case where the second landing pad has negative coordinates and the distance between them is more than the distance threshold
+        """
+        obj1 = object_in_world.ObjectInWorld.create(0, 0, 0)[1]
+        obj2 = object_in_world.ObjectInWorld.create(-2, -2, 0)[1]
+        expected = False
+
+        actual = landing_pad_tracking.LandingPadTracking._LandingPadTracking__is_similar(obj1, obj2, DISTANCE_SQUARED_THRESHOLD)
+
+        assert actual == expected
+
+    # pylint: enable=protected-access
+
+class TestMarkFalsePositive:
+    """
+    Test if landing pad tracking correctly marks a detection as a false positive
+    """
+    # Required for testing
+    # pylint: disable=protected-access
+    def test_mark_false_positive_no_similar(self, tracker: landing_pad_tracking.LandingPadTracking, detections1: "list[object_in_world.ObjectInWorld]"):
+        """
+        Test if marking false positive adds detection to list of false positives
+        """
+        false_positive = object_in_world.ObjectInWorld.create(20, 20, 20)[1]
+        tracker._LandingPadTracking__unconfirmed_positives = detections1
+        expected = [false_positive]
+        expected_unconfirmed_positives = [detections1[0], detections1[1], detections1[2], detections1[3], detections1[4]]
+        
+        tracker.mark_false_positive(false_positive)
+
+        assert tracker._LandingPadTracking__false_positives == expected
+        assert tracker._LandingPadTracking__unconfirmed_positives == expected_unconfirmed_positives
+
+    def test_mark_false_positive_with_similar(self, tracker: landing_pad_tracking.LandingPadTracking, detections2: "list[object_in_world.ObjectInWorld]"):
+        """
+        Test if marking false positive adds detection to list of false positives and removes similar landing pads
+        """
+        false_positive = object_in_world.ObjectInWorld.create(1, 1, 1)[1]
+        tracker._LandingPadTracking__unconfirmed_positives = detections2
+        expected = [false_positive]
+        expected_unconfirmed_positives = [detections2[2], detections2[3], detections2[4]]
+        
+        tracker.mark_false_positive(false_positive)
+
+        assert tracker._LandingPadTracking__false_positives == expected
+        assert tracker._LandingPadTracking__unconfirmed_positives == expected_unconfirmed_positives
+    
+    def test_mark_multiple_false_positive(self, tracker: landing_pad_tracking.LandingPadTracking, detections1: "list[object_in_world.ObjectInWorld]"):
+        """
+        Test if marking false positive adds detection to list of false positives
+        """
+        false_positive1 = object_in_world.ObjectInWorld.create(20, 20, 1)[1]
+        false_positive2 = object_in_world.ObjectInWorld.create(40, 40, 1)[1]
```

Comment:
We would want them to overlap some of the (different) unconfirmed landing pads to make sure they are removed correctly.